### DISCUSSION TITLE
Adding opened tabs to saved groups or windows

### DISCUSSION THREAD

**Discussion by FreePDF64** [okaryo/TabTabTab#361] [Ideas]
The possibility to add opened tabs to my saved groups/windows would be very fine as well as the possibility to open ONE SELECTED tab from a saved group would be fine too.

**Comment by okaryo** [comment]
@FreePDF64 
Certainly, that's a great idea! Additionally, allowing the deletion of tabs or groups from saved groups/windows seems like a good enhancement.
However, I'm currently unsure how to implement the feature for adding opened tabs to saved groups/windows from a UI perspective. If I come up with a viable implementation, I would be grateful if you could assist in testing it.

**Comment by FreePDF64** [comment]
Anytime my friend anytime.

**Comment by FreePDF64** [comment]
Hello Ryotaro Oka,

Unfortunately, I haven't heard from you in a long time. I hope everything is ok on your side and you and your familiy are healthy and feel well.


Are you going to continue developing TabTabTab, or have you allowed yourself a break from creation? Understandably, from my point of view.

I am very much looking forward to your answer.

Kind regards,
Michael

________________________________
Von: Ryotaro Oka ***@***.***>
Gesendet: Sonntag, 7. April 2024 16:44
An: okaryo/TabTabTab ***@***.***>
Cc: Michael ***@***.***>; Mention ***@***.***>
Betreff: Re: [okaryo/TabTabTab] Adding opened tabs to saved groups or windows (Discussion #361)


@FreePDF64<https://github.com/FreePDF64>
Certainly, that's a great idea! Additionally, allowing the deletion of tabs or groups from saved groups/windows seems like a good enhancement.
However, I'm currently unsure how to implement the feature for adding opened tabs to saved groups/windows from a UI perspective. If I come up with a viable implementation, I would be grateful if you could assist in testing it.

**Reply by okaryo** [reply]
Hello Michael,

Thank you for your message. I apologize for not progressing on the development of TabTabTab. I haven't forgotten about this new feature, but things have been hectic since I left my company recently, and I haven't been able to find the time to work on the extension. It looks like it will take a bit more time before I can get back to developing. I appreciate your patience.

Best regards,

Ryotaro

**Reply by okaryo** [reply]
Hello Michael,

I wanted to let you know that the feature you requested has been implemented in v2.33.0. You can now open or delete tabs and groups from saved windows/groups. I apologize for the delay in getting this done.

The update is currently under review in the store, and it will be available once the review is complete. I’d appreciate it if you could give it a try, and let me know if you find any issues or areas that need improvement.

Best regards,
Ryotaro

**Comment by FreePDF64** [comment]
Hello Ryotaro,

Thank you so much for these new functions. You have done a great job implementing them! Other users will be impressed too.
I already use these new functions intensively on my work PC.

Do you think that it is possible to add tabs to a saved window or group?

And please: You have all the time in the world to add more functions. Your private life and work come first.

Best regards,
Michael

**Reply by okaryo** [reply]
Hi Michael,

Thank you for trying out the new features so quickly!

I believe adding tabs to saved windows or groups is possible. In that case, would features like drag-and-drop reordering within saved windows or groups, or duplicating windows, groups, or tabs be useful as well? I’ll start by working on the tab addition feature, so if you think those other features might be helpful, feel free to let me know.

Thanks for your concern! Developing TabTabTab is actually a way for me to relax and refresh, and improving it benefits me as a user too, so no worries. I appreciate your support!

Best regards,
Ryotaro

**Reply by FreePDF64** [reply]
Hi,

"would features like drag-and-drop reordering within saved windows or groups, or duplicating windows, groups, or tabs be useful as well?"

Oh yes! This would be very wonderful.


Thank you,
Michael

**Comment by okaryo** [comment]
@FreePDF64 
Hi,

I've added the feature to add tabs to saved windows/groups! This is included in v2.35.0, which is currently under review in the store. Once it's available, I hope you’ll give it a try. Let me know anytime if you run into any issues.

#414 

Sorry for the slow development recently. I’m working on other features little by little, so thank you for your patience. (As a user of TabTabTab myself, I’m also excited to see it become even more useful!)

**Comment by FreePDF64** [comment]
Hi,

Thank you for this new nice feature. It works very well.

Nevertheless I found an issue when you edit the group name:

***@***.***

I use Windows 11.

Thx!
Michael

Von: Ryotaro Oka ***@***.***>
Gesendet: Sonntag, 1. Dezember 2024 12:55
An: okaryo/TabTabTab ***@***.***>
Cc: Michael ***@***.***>; Mention ***@***.***>
Betreff: Re: [okaryo/TabTabTab] Adding opened tabs to saved groups or windows (Discussion #361)


@FreePDF64<https://github.com/FreePDF64>
Hi,

I've added the feature to add tabs to saved windows/groups! This is included in v2.35.0, which is currently under review in the store. Once it's available, I hope you’ll give it a try. Let me know anytime if you run into any issues.

#414<https://github.com/okaryo/TabTabTab/pull/414>

Sorry for the slow development recently. I’m working on other features little by little, so thank you for your patience. (As a user of TabTabTab myself, I’m also excited to see it become even more useful!)

**Reply by FreePDF64** [reply]
Hi,

I experience misconduct here from time to time:

For example, if I add the following website - https://www.ndr.de/ndr2/live/stationndrzwei102-radioplayer.html (it's a radio live stream)
then the automatic does not work here

![image](https://github.com/user-attachments/assets/0d7c995a-374a-4b19-9dce-f9ca0a583551)

This behavior occurs on various sites - especially with radio live streams... Wouldn't it be better if we could give it a name yourself?

**Reply by okaryo** [reply]
Thank you for reporting this. I’ve confirmed the issue on my end and will fix it soon. I’ll also add a feature to allow manual naming when such errors occur.

**Reply by okaryo** [reply]
It has been fixed in version 2.37.1, which is currently under review in the store.
Additionally, manual title input is now available when an error occurs.

**Comment by FreePDF64** [comment]
Hi,

Thank you for your effort to make TabTabTab the best Tab Manager for Chromium browsers!!

I found an issue when you edit the group title. You have to click inside and see:

![thumbnail_image](https://github.com/user-attachments/assets/82513c1f-7602-4673-96b4-f8f483a9fdde)


Kind regards,
Michael
________________________________
Von: Ryotaro Oka ***@***.***>
Gesendet: Dienstag, 17. Dezember 2024 05:07
An: okaryo/TabTabTab ***@***.***>
Cc: Michael ***@***.***>; Mention ***@***.***>
Betreff: Re: [okaryo/TabTabTab] Adding opened tabs to saved groups or windows (Discussion #361)


It has been fixed in version 2.37.1, which is currently under review in the store.
Additionally, manual title input is now available when an error occurs.

**Reply by okaryo** [reply]
Thank you for reaching out and for your kind words about TabTabTab!

Regarding the issue you mentioned, I couldn't view the image or content referenced as "cid:fd782ced-2a56-4ed3-aa72-a6fe2a383e30." Could you please provide more details or reattach the image if possible?

**Reply by FreePDF64** [reply]
Hi,

I updated the picture above. I hope you can see it and you can reproduce the error: go to a saved group and edit the title of the group.

**Reply by okaryo** [reply]
Thanks! I'll fix it in the next release.

**Reply by okaryo** [reply]
I’ve fixed this issue in v2.37.2, which is currently under review in the store. Thank you for reporting it!
